example/contracts_generator.yaml

```diff
@@ -1,5 +1,20 @@
-# Path to contracts' .csproj project file (required)
-input: ../contractsgenerator/examples/project/aggregated/Combined/Combined.csproj
+# Possible input methods, specify exactly one (required)
+input:
+  # Array of paths pointing to .csproj contracts project files
+  # project:
+  #   - ../contractsgenerator/examples/project/aggregated/Combined/Combined.csproj
+  # Path to a single source file
+  # file: ExampleContracts/Auth.cs
```

Comment:
I _think_ you won't need it, as this was created for testing mainly.

Reply:
By the way, isn't `generate.sh file -i some/file.cs` and `generate.sh path -i some/file.cs` equivalent? Perhaps the whole `file` subcommand can be removed to avoid confusion

Reply:
They are, although they use slightly different ways of finding things that is (was?) beneficial for preliminary testing. Will probably remove soon.